examples/Cifar10DataTensorFlow.py

```diff
@@ -0,0 +1,39 @@
+import tensorflow as tf
+from aperturedb.TensorFlowData import TensorFlowData
+from typing import List, Tuple
+from PIL import Image
+import io
+
+
+class Cifar10DataTensorFlow(TensorFlowData):
+    """
+    **ApertureDB ingestable Dataset, which is sourced from
+    [Cifar10 (tensorflow.datasets)](https://www.tensorflow.org/datasets/catalog/cifar10)**
+    """
+
+    def __init__(self):
+        (x_train, y_train), (x_val, y_val) = tf.keras.datasets.cifar10.load_data()
+        x = tf.concat([x_train, x_val], axis=0)
+        y = tf.concat([tf.squeeze(y_train), tf.squeeze(y_val)], axis=0)
+        ds = tf.data.Dataset.from_tensor_slices((x, y))
+        super().__init__(ds)
+
+    def __image__from__cifar10__(self, arr):
+        bytes = io.BytesIO()
+        Image.fromarray(arr, 'RGB').save(bytes, format='JPEG')
+        return bytes.getvalue()
+
+    def generate_query(self, idx: int) -> Tuple[List[dict], List[bytes]]:
+        x, y = self.loaded_dataset[idx]
+        img_ref = (idx % 99998) + 1
+        q = [{
+            "AddImage": {
+                "_ref": img_ref
+            }
+        }]
+        q[0]["AddImage"]["properties"] = {
+            "label": str(y.numpy()),
+            "stage": "train" if idx < 50000 else "val"
```

Comment:
CIFAR-10 is documented as having 50,000 training images and 10,000 test images, but why don't we get this magic number from `x_train` in `__init__`?  And why are we labelling test images as `val`?

Reply:
This was just following the guideline in this prior notebook that was built. I think it might be better to stick to the conventions of the dataset. will change the notebook, instead of this class. Also, will drop the magic number.